ipn/ipnlocal/local.go

```diff
@@ -1379,13 +1383,13 @@ func (b *LocalBackend) Start(opts ipn.Options) error {
 			// prevent it from restarting our map poll
 			// HTTP request (via doSetHostinfoFilterServices >
 			// cli.SetHostinfo). In practice this is very quick.
-			t0 := time.Now()
-			timer := time.NewTimer(time.Second)
+			t0 := b.clock.Now()
+			timer, timerChannel := b.clock.NewTimer(time.Second)
 			select {
 			case <-b.gotPortPollRes:
 				b.logf("[v1] got initial portlist info in %v", time.Since(t0).Round(time.Millisecond))
```

Comment:
time.Since here is mixing clocks

Reply:
replaced w/ `clock.Now().Sub(t0)`

Reply:
can we add a `Since` method instead to keep the code looking the same?